examples/stream-in/progress.js

```diff
@@ -0,0 +1,39 @@
+const { EventEmitter } = require('events');
```

Comment:
```suggestion
'use strict';
const { EventEmitter } = require('events');
```

since this is plain JS?